playbooks/rhv/openshift-cluster/register-nodes.yml

```diff
@@ -0,0 +1,12 @@
+---
+- name: Subscribe nodes
+  hosts:
+    - nodes
+    - lb
```

Comment:
Please add "if lb defined" in case user does not request lb

Reply:
I don't think that's necessary.

Reply:
ok, Will it fail if lb in undefined?

Reply:
Actually, I'm going to remove this playbook; prerequisites.yml invalidates the need for it.

Reply:
+1